compiler/MetadataUtils.ts

```diff
@@ -370,6 +370,9 @@ export default class MetadataUtils {
         // https://github.com/AdguardTeam/PopupBlocker/issues/241
         '*://epicgames.com/*',
         '*://*.epicgames.com/*'
+        // https://github.com/AdguardTeam/AdguardFilters/issues/90068
```

Comment:
should be added in two places in this file(see epicgames.com)

Reply:
Done.